### DISCUSSION TITLE
de/1-09-further-information/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#20] [General]
# de/1-09-further-information/

https://ai.lernos.org/de/1-09-further-information/

**Comment by simondueckert** [comment]
Eine Sektion mit Studien wäre noch hilfreich, z.B. Evidenz für Steigerung von Produktivität/Kreativität (z.B. [Stanford/MIT Studie](https://www.forbes.com/sites/johanmoreno/2023/04/25/in-a-real-world-study-ai-boosts-worker-productivity-by-14/amp/), Verbreitung des Einsatzes von LLM in Unternehmen, Marktanteile ChatGPT vs. Copilot. etc.

**Comment by simondueckert** [comment]
Habe den t3n Podcast [Hat KI irgendwann ein Bewusstsein?](https://t3n-podcast.podigee.io/593-karsten-wendland) mit [Karsten Wendland](https://www.linkedin.com/in/karstenwendland/) gehört. Er macht den Podcast [Selbstbewusste KI](https://www.ki-bewusstsein.de/podcast), den sollten wir auf jeden Fall im Leitfaden aufnehmen.

**Comment by simondueckert** [comment]
Den [Microsoft WorkLab Podcast](https://podcasts.apple.com/de/podcast/worklab/id1582908812) könnten wir noch ergänzen.

**Comment by simondueckert** [comment]
Buchtipp: [Wie Maschinen lernen - Künstliche Intelligenz verständlich erklärt](https://link.springer.com/book/10.1007/978-3-658-26763-6).

**Comment by Flesch-04-18** [comment]
Wie wäre ein Link zum EU AI Act?

**Comment by rednaxelaesor** [comment]
Weitere KI-Podcast's
https://neuzeit.podigee.io/feed/mp3
https://anchor.fm/s/34520fb0/podcast/rss

**Comment by KathaZ** [comment]
Tolle Übersicht, vielen Dank!

**Comment by simondueckert** [comment]
https://www.telekom.com/de/medien/podcasts/explained-human-views-on-ai

**Comment by simondueckert** [comment]
Der [KI-Podcast des BR](https://origin.podcast.br.de/mediathek/podcast/der-ki-podcast/alle/898) mit Fritz und Gregor müssen wir auch in der nächsten Version mit aufnehmen.

**Reply by rednaxelaesor** [reply]
#slopaganda
